less/admin/PermissionsPage.less

```diff
@@ -49,19 +50,36 @@
     color: @muted-color;
   }
   thead th {
+    position: -webkit-sticky;
```

Comment:
I don't suppose we can avoid vendor specific stuff here?

Reply:
for now we need it for some old safari browsers